lib/Ethereum/RPC/Client.pm

```diff
@@ -38,13 +38,16 @@ sub AUTOLOAD {
         method => $method,
         params => (ref $_[0] ? $_[0] : [@_]),
     };
-
+    
+    # dies on connection problems
```

Comment:
The comment is for the wrong statement.

Reply:
This comment indicates that `http_client->post->result` will issue a `die` if there are any connection problems. which statement should I put this comment on?

Reply:
I thought this points to https://github.com/binary-com/perl-Ethereum-RPC-Client/blob/c02420d4279c3342b8dbf13b8748e2845b0bed11/lib/Ethereum/RPC/Client.pm#L47, so if it will die within itself:
```suggestion
    # Will issue 'die' itself on connection problems
```